contents/handbook/growth/growth-engineering/growth-sessions.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Growth sessions
+sidebar: Handbook
+showTitle: true
+---
+
+Whilst the majority of our focus is making the product more useful, these sessions make sure we're not missing anything obvious from a pure growth perspective.
+
+They have led to some significant increases in monthly revenue growth.
+
+!> Disclaimer: these are worth doing post product market fit. Before then, just focus on making something people want, rather than optimizing.
+
+## What it is
+
+### Before the meeting
+* Before the meeting, everyone creates a list of hypotheses. These can be problems or opportunities. They should all be growth engineering related, and they should be focused on improving our long term monthly growth rate.
+* Examples:
+    * "More expensive new pricing has hurt signups"
+    * "We're not solving bugs quickly and this is hurting word of mouth growth"
+    * "We are losing b2c customers to open source"
+    * "We should charge for products separately so we can upsell on features rather than volume. This would let us charge sooner than our current 1m events/month limit, and would let us keep prices low for people that don't use the full breadth of the platform"
+
+### When the meeting starts
+* When the meeting starts, allocate 10 minutes for people to read through everything and to add any hypotheses if they've not prepared.
+* When the meeting has started, watch some session recordings of users going through key flows of the product. This may generate some further hypotheses. For example, users converting to paying customers, users activating, users inviting their colleagues.
```

Comment:
```suggestion
* After reading, watch session recordings of users going through key flows of the product. Allocate 20 minutes for this. This may generate some further hypotheses. For example, users converting to paying customers, users activating, users inviting their colleagues.
```